great_expectations/expectations/metrics/column_aggregate_metric_provider.py

```diff
@@ -60,7 +60,7 @@ def column_aggregate_value(
     if issubclass(engine, PandasExecutionEngine):
 
         def wrapper(metric_fn: Callable):
-            @metric_value(engine=PandasExecutionEngine)
+            @metric_value(engine=engine)
```

Comment:
The second part of the bug was hard coding the engine names in the metric decorators. I've grepped all the metric provider files and this file is the only one I saw containing hard coded values.